src/content/docs/faqs/common-questions-for-developers.md

```diff
@@ -1,6 +1,74 @@
 ---
-title:  "Common Questions for Developers"
-description:  "Common Questions for Developers"
+title: "Developers FAQ"
+description: "Frequently Asked Questions tailored for developers working with Angor."
 sidebar:
-  order: 3
+  order: 4
 ---
+
+## **What is Angor, and why is it relevant for developers?**
+Angor is a decentralized protocol built on Bitcoin and Nostr. It offers developers the opportunity to contribute to an open-source project that empowers secure crowdfunding without intermediaries.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How can I contribute to Angor’s development?**
```

Comment:
I am not sure this is the path we want to go down when we talk to developers.
Angor does not need contributors really we have enough devs to build Angor.
What we want is dev to use Angors platform to raise funds for their projects.
We should address developers on that approach.